src/patternfly/components/Sequence/sequence.scss

```diff
@@ -0,0 +1,374 @@
+
+  // Content Status Color
+  --pf-c-sequence__content-status--Color: var(--pf-global--Color--200);
+
+  // Content Status Font
+  --pf-c-sequence__content-status--FontSize: var(--pf-global--FontSize--xs);
+
+  // Modifier - current link
+  --pf-c-sequence__link--m-current--BorderBottomHeight: var(--pf-global--BorderWidth--lg);
+  --pf-c-sequence__link--m-current--BorderBottomColor: var(--pf-global--primary-color--100);
+
+  // Modifier - current color
+  --pf-c-sequence__number--m-current--Color: var(--pf-global--primary-color--100);
+  --pf-c-sequence__content-title--m-current--Color: var(--pf-global--primary-color--100);
+
+  // Modifier - current font
+  --pf-c-sequence__number--m-current--FontSize: var(--pf-global--FontSize--xl);
+  --pf-c-sequence__content-title--m-current--FontWeight: var(--pf-global--FontWeight--regular);
+
+  // Modifier - current number
+  --pf-c-sequence__number--m-current--MarginRight: var(--pf-global--spacer--md);
+  --pf-c-sequence__number--m-current--MinWidth: var(--pf-global--spacer--xl);
+  --pf-c-sequence__number--m-current--MaxHeight: var(--pf-global--spacer--xl);
+  --pf-c-sequence__number--m-current--BorderColor: var(--pf-global--primary-color--100);
+
+  position: relative;
+  display: flex;
+  justify-content: center;
+  background-color: var(--pf-c-sequence--BackgroundColor);
+
+  // Hide scrollbars
+  &::-webkit-scrollbar {
+    display: none;
+  }
+}
+
+.pf-c-sequence__control {
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  padding:
+    var(--pf-c-sequence__control--PaddingTop)
+    var(--pf-c-sequence__control--PaddingRight)
+    var(--pf-c-sequence__control--PaddingBottom)
+    var(--pf-c-sequence__control--PaddingLeft);
+  border-top: var(--pf-c-sequence__control--BorderTopHeight) solid var(--pf-c-sequence__control--BorderTopColor);
+  border-bottom: var(--pf-c-sequence__control--BorderBottomHeight) solid var(--pf-c-sequence__control--BorderBottomColor);
+
+  &.pf-m-hover,
+  &:hover {
+    border-top: var(--pf-c-sequence__control--hover--BorderTopHeight) solid var(--pf-c-sequence__control--hover--BorderTopColor);
+    border-bottom: var(--pf-c-sequence__control--hover--BorderBottomHeight) solid var(--pf-c-sequence__control--hover--BorderBottomColor);
+  }
+
+  &.pf-m-active,
+  &:active {
+    border-top: var(--pf-c-sequence__control--active--BorderTopHeight) solid var(--pf-c-sequence__control--active--BorderTopColor);
+    border-bottom: var(--pf-c-sequence__control--active--BorderBottomHeight) solid var(--pf-c-sequence__control--active--BorderBottomColor);
+  }
+
+  &.pf-m-focus,
+  &:focus {
+    border-top: var(--pf-c-sequence__control--focus--BorderTopHeight) solid var(--pf-c-sequence__control--focus--BorderTopColor);
+    border-bottom: var(--pf-c-sequence__control--focus--BorderBottomHeight) solid var(--pf-c-sequence__control--focus--BorderBottomColor);
+  }
+}
+
+.pf-c-sequence__control-left {
```

Comment:
instead of `left` and `right`, I would use a more [logical property-esque](https://www.w3.org/TR/css-logical-1/) name like `start` and `end`

Reply:
And I wonder if this should just be `.pf-c-sequence__control`, since they're the same element with different presentations, then use `.pf-m-start` and `.pf-m-end` as modifiers. Similar to the `.pf-c-switch__label` element in the Switch component.

Reply:
sounds good!